src/ui/NextUI.fs

```diff
@@ -11,9 +11,43 @@ type NextUI =
         : ReactElement =
         React.imported ()
 
-    static member Component (name: string) (xs: IReactProperty list) : ReactElement =
-        Interop.reactApi.createElement (import name "@nextui-org/react", createObj !!xs)
+    static member inline color (colorName: string) = prop.custom("color", colorName)
```

Comment:
This is good. More like this as needed. I think any time we use `prop.custom` on the same key more than once, it should get one of these instead.

Reply:
Okay!